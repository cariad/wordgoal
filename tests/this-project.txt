[33mtests/documents/smoke.md[39m [32mâ–‹                                              [39m [94m8[39m / [94m600[39m
[33mREADME.md[39m                [32mâ–Œ                                              [39m [94m7[39m / [94m600[39m

