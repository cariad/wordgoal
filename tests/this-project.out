[33mtests/data/no_front_matter.md[39m      [32m‚ñé                                [39m [94m  8[39m / [94m1,000[39m
[33mtests/data/partial_front_matter.md[39m [32m‚ñç                                [39m [94m 12[39m / [94m1,000[39m
[33mtests/data/plus_front_matter.md[39m    [32m‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñå     [39m [94m 10[39m / [94m   12[39m
[33mtests/data/smoke.md[39m                [32m‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç      [39m [94m  8[39m / [94m   10[39m
[33mtests/data/smoke.txt[39m               [32m‚ñé                                [39m [94m  8[39m / [94m1,000[39m
[33mREADME.md[39m                          [32m‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ               [39m [94m326[39m / [94m  600[39m
